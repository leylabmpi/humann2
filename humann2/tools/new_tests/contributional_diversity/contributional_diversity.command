# basic prevalence filter
humann2_contributional_diversity \
	     -i contributional_diversity.input \
	     -o contributional_diversity.output \

