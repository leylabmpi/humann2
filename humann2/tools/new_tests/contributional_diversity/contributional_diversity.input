#	1	2	3	4
A	11	22	33	0
A|1	10	20	30	0
A|2	0	0	0	0
A|unclassified	1	2	3	0
B	1	1	1	0
B|1	1	0.5	0	0
B|2	0	0.5	1	0
C	3	3	3	0
C|1	1	1	1	0
C|2	1	1	1	0
C|unclassified	1	1	1	0
