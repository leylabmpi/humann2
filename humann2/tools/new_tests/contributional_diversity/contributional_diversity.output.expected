# Feat \ Stat	1: Prevalence (N)	2: Prevalence (frac)	3: Mean abundance	4: Mean explained (frac)	5: Alpha contrib. div.	6: Beta contrib. div.
A	3	0.75	22	0.909091	0	0
B	3	0.75	1	1	0.166667	0.666667
